src/wire/igmp.rs

```diff
@@ -173,7 +173,7 @@ impl<T: AsRef<[u8]> + AsMut<[u8]>> Packet<T> {
 #[derive(Debug, PartialEq, Eq, Clone)]
 pub enum Repr {
     MembershipQuery {
-        max_resp_time: Duration,
+        max_resp_duration: Duration,
```

Comment:
Rebase error? I specifically renamed it to `max_resp_time`, since that's how the field is called in the RFC.

Reply:
I thought otherwise, because in `Repr` we're no longer dealing with the `max_resp_time` code but with a proper `Duration`. I'll rename it back if you don't agree that that's a significant difference between the two.

Reply:
The *concept* is the same (maximum response time), the *units* are different (floating point in i8 vs Duration). You could call it `max_resp_time_fp` and `max_resp_time_duration` but that's annoyingly long.

Reply:
Ok!